item-impl/src/main/java/com/example/auction/item/impl/PItemEntity.java

```diff
@@ -51,7 +52,7 @@ private Behavior created(PItemState state) {
 
         builder.setCommandHandler(StartAuction.class, (start, ctx) -> {
             if (start.getUserId().equals(state().getItem().get().getCreator())) {
-                return ctx.thenPersist(new AuctionStarted(entityUuid(), Instant.now()), evt -> ctx.reply(Done.getInstance()));
+                return ctx.thenPersist(new AuctionStarted(entityUuid(), Instant.now()), done(ctx));
```

Comment:
It appears that `ctx.done` already exists! http://www.lagomframework.com/documentation/1.2.x/java/api/com/lightbend/lagom/javadsl/persistence/PersistentEntity.CommandContext.html#done--

However it doesn't do the same thing.